Source/WTF/WTF_Private.modulemap

```diff
@@ -0,0 +1,40 @@
+framework module WTF_Private [system] {
+    explicit module Platform {
+        header "Platform.h"
+        private textual header "PlatformCallingConventions.h"
+        private textual header "PlatformCPU.h"
+        private textual header "PlatformEnable.h"
+        private textual header "PlatformEnableCocoa.h"
+        private textual header "PlatformHave.h"
+        private textual header "PlatformLegacy.h"
+        private textual header "PlatformOS.h"
+        private textual header "PlatformRegisters.h"
+        private textual header "PlatformUse.h"
+
+        export *
+    }
+
+    explicit module Compiler {
+        header "Compiler.h"
+    }
+
+    explicit module spi {
+        umbrella "PrivateHeaders/spi"
```

Comment:
Also I'm a little leery of nested umbrella directories but I guess if it works then ok...